### ISSUE TITLE
.json format custom dataset

### ISSUE THREAD

**Issue by akmaralAW** [yassouali/CCT#18]
Hello !

Great job ! Thanks for sharing it.

I was wondering is it possible to run it with .json format annotations ?

**Comment by yassouali** [comment]
Hi @akmaralAW 
Thank you for your interest;

The method can work with any type of pixel-wise annotations, in this case, all you'll need is to write your custom dataset and replace the existing one.

**Comment by akmaralAW** [comment]
Hello !
Thanks for your quick response. I guess ground truth is .png/jpg images, as shown in your original paper.

**Comment by yassouali** [comment]
@akmaralAW 

Yes, but I suppose if your segmentation maps are in the form of a json file (like COCO for example), you can just use a custom dataloader to load them and train the network.

**Comment by akmaralAW** [comment]
I am training with .png ground truth labels. So far it is going well. Usually in segmentation tasks one has to assign RGB values of labels. In your case there is no mention about label colours.
